Framework/Core/include/Framework/ASoA.h

```diff
@@ -1194,6 +1194,11 @@ class Filtered : public T
     return mSelection->GetNumSlots();
   }
 
+  int64_t tableSize() const
```

Comment:
Why not returning `unfiltered()` and using `size()` from it?

Reply:
@ktf I've rewrote it through "asArrowTable()"